admin-dev/themes/new-theme/js/pages/order/index.js

```diff
@@ -34,11 +34,34 @@ import SubmitBulkExtension from '../../components/grid/extension/submit-bulk-act
 import BulkActionCheckboxExtension from '../../components/grid/extension/bulk-action-checkbox-extension';
 import FiltersSubmitButtonEnablerExtension
   from '../../components/grid/extension/filters-submit-button-enabler-extension';
-import ModalFormSubmitExtension from "../../components/grid/extension/modal-form-submit-extension";
+import ModalFormSubmitExtension from '../../components/grid/extension/modal-form-submit-extension';
+import PreviewExtension from '../../components/grid/extension/preview-extension';
 
 const $ = window.$;
 
 $(() => {
+  // @todo: I think previewRenderer() should accept `id` and some kind of `params` argument,
+  // so grid can pass other data (like URL) to preview renderer
+  // in case it uses ajax to load content.
+  // @todo: Extract into separate file.
+  function previewRenderer(id) {
+    console.log($('.preview-toggle').data('preview-data-url'));
+    return new Promise((resolve, reject) => {
+      $.ajax({
+        url: $('.preview-toggle').data('preview-data-url'),
```

Comment:
URL should be passed as function argument, otherwise, how do you know it takes correct URL?